.github/workflows/base-component-descriptor.yaml

```diff
@@ -26,6 +26,23 @@ on:
         required: false
         type: string
         default: SAP SE
+      component-references:
+        required: false
+        type: string
+        description: |
+          relative path (to repo root) to YAML/Json file containing definition of
+          component-references added to base-component-descriptor, defaults to
+          `.ocm/component-references.yaml`.
+          action will only raise on absence of file if an explicit value is given.
+
+          Example contents:
+          - componentName: github.com/gardener/gardener
+            version: v1.110.0
+            name: gardener/gardener
+            # following key-values pairs are optional
+            digest: foo
+            extraIdentity:
+              foo: bar
```

Comment:
likewise - while this extra-identity is syntactically correct, I think it will not help too much. Consider especially that within the entire gardener-bom we have not a single occurrance of a cref w/ an extra-id (also, I doubt there will very likely be any such usage _ever_ - but that's a different story). So, also to keep things simple, and make the example less confusing, I would suggest to drop it (people should _understand_ that they can add additional attributes into a YAML file :-)